client/card-readers/index.js

```diff
@@ -0,0 +1,52 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import Page from 'components/page';
+import ReadersList from './list';
+import ReadersSettings from './settings';
+import { TabPanel } from '@wordpress/components';
+
+import './style.scss';
+
+export const ConnectedReaders = () => {
+	return (
+		<Page>
+			<TabPanel
+				className="wcpay-card-readers-page"
+				activeClass="active-tab"
+				tabs={ [
+					{
+						name: 'connected-readers',
+						title: __(
+							'Connected readers',
+							'woocommerce-payments'
+						),
+						className: 'connected-readers-list',
+					},
+					{
+						name: 'receipt-details',
+						title: __( 'Receipt details', 'woocommerce-payments' ),
+						className: 'connected-readers-receipt-details',
+					},
+				] }
+			>
+				{ ( tab ) => {
+					if ( 'receipt-details' === tab.name ) {
+						return <ReadersSettings />;
```

Comment:
IMHO, "reader settings" sounds more like the internal configuration of a card reader itself. In our case, we want to only display settings to configure its receipt. Does a name such as `ReceiptSettings` sound more apt here?

Reply:
Makes sense, updated.